src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -17,6 +20,13 @@ public final class HearingsWindowMapping {
     public static final int DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED = 28;
     public static final int DAYS_TO_ADD_HEARING_WINDOW_TODAY_POSTPONEMENT = 14;
     public static final int DAYS_TO_ADD_HEARING_WINDOW_TODAY = 1;
+    public static final int DAYS_TO_ADD_HEARING_WINDOW_TODAY_ADJOURNMENT = 14;
+    public static final String DATE_TO_BE_FIXED = "Date to be fixed";
+    public static final String FIRST_AVAILABLE_DATE = "First available date";
+    public static final String FIRST_AVAILABLE_DATE_AFTER = "First available date after";
+    public static final String PROVIDE_PERIOD = "Provide period";
+    public static final String PROVIDE_DATE = "Provide date";
+
```

Comment:
Remove extra whitespace (also some extra after the constructor which could be removed)

Reply:
done